app/src/main/res/values-af/strings.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources></resources>
```

Comment:
Do we really want/need to push empty translations? Is there no switch in Weblate to prevent that? Ideally, we only push translations once they are (almost) complete.

Reply:
Not that I know of, we've been doing this so far for all other projects.